modules/nw-udn-cr.adoc

```diff
@@ -34,32 +34,89 @@ spec:
     mtu: 9000 # <5>
     subnets:
       - "10.0.0.0/24"
-      - "2001:db8::/64" # <6>
-    JoinSubnets: [] # <7>
-    IPAMLifecycle: Persistent # <8>
+      - "2001:db8::/60" # <6>
 ----
 <1> Name of your `UserDefinedNetwork` resource. This should not be `default` or duplicate any global namespaces created by the Cluster Network Operator (CNO).
-<2> The `topology` field describes the network configuration, accepted values are `Layer2` and `Layer3`. `Layer3` topology creates a layer 2 segment per node, each with a different subnet. Layer 3 routing is used to interconnect node subnets. `Layer2` topology creates one logical switch shared by all nodes.
+<2> The `topology` field describes the network configuration, accepted values are `Layer2` and `Layer3`. Specifying a `Layer2` topology type creates one logical switch that is shared by all nodes.
```

Comment:
```suggestion
<2> The `topology` field describes the network configuration. Accepted values are `Layer2` and `Layer3`. Specifying a `Layer2` topology type creates one logical switch that is shared by all nodes.
```

Probably a period or a conjunction.

Reply:
Went with a semi-colon since the sentences are so short.